agent_api_rest/src/lib.rs

```diff
@@ -21,31 +22,76 @@ use offers::offers;
 pub const AGGREGATE_ID: &str = "agg-id-F39A0C";
 
 pub fn app(state: ApplicationState<IssuanceData, IssuanceDataView>) -> Router {
+    let base_path = config!("base_path").map(|mut base_path| {
```

Comment:
Can this be extracted in its own helper function? I feel like it clutters the `app()` function.